### PR TITLE
Fix StdConfig#setCmdLineArgs (#509)

### LINKED ISSUE #509: StdConfig setCmdLineArgs behaves as 'add' not 'set'
**Describe the bug**
Calling setCmdLineArgs will only add cmdLine Args, it does not replace the existing ones.
Based on the name and documentation, this is incorrect - it should remove existing args.

Also, the unit test (StdConfigTest) should be updated to check this condition.

### PR DESCRIPTION
**Pull request by alex-kar**
Fixes #509

* Update setter to remove previously set arguments.
* Cover with test.
* Update javadoc.

### All Submissions:

Have you checked that...
* [+] Your pull request is to the *_main_* branch
* [+] Your code is up to date with the latest code from the *_main_*
* [+] You followed the guidelines in our [Developer Guide](https://sites.google.com/view/andhow/developer)?
* [+] Your code does not decrease test coverage (maybe it improves coverage!!)
* [+] If this is related to an issue, please include a link to the issue with the 'Fixes #XXX' syntax.

### CODE REVIEW

File: andhow-core/src/test/java/org/yarnandtail/andhow/StdConfigGetterAndSetterTest.java [hunk 1]
```diff
@@ -237,14 +237,37 @@ public void FixedValuesBasedOnNamesTest() {
...
+
+		// Set empty array
+		config.setCmdLineArgs(new String[0]);
+
+		actualArgs = config.getCmdLineArgs();
+
+		assertEquals(0, actualArgs.size());
+
+		// Set null
+		config.setCmdLineArgs(new String[0]);
```
Comment (eeverman) [inline andhow-core/src/test/java/org/yarnandtail/andhow/StdConfigGetterAndSetterTest.java:266]: Oops, this test doesn't test the null.  Also, to really test it, it should initially be set non-empty.

**Review by eeverman — CHANGES_REQUESTED** [review]
Nice fix!  One minor comment on the tests.
Thanks again!

File: andhow-core/src/test/java/org/yarnandtail/andhow/StdConfigGetterAndSetterTest.java (hunk 1 above)
Comment (alex-kar) [reply andhow-core/src/test/java/org/yarnandtail/andhow/StdConfigGetterAndSetterTest.java]: Fixed

**Review by eeverman — APPROVED** [review]
Looks great!
